packages/Checkbox/src/Checkbox.styles.js

```diff
@@ -0,0 +1,176 @@
+import { css } from "styled-components";
+import { toInt, lineHeightValue, z } from "@paprika/stylers/lib/helpers";
+import { boxSizingStyles, visuallyHidden } from "@paprika/stylers/lib/includes";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import tokens from "@paprika/tokens";
+
+const getLabelLeftPadding = (checkboxSize, hasLabel) => {
+  return hasLabel ? `${toInt(checkboxSize) + toInt(tokens.space)}px` : checkboxSize;
+};
+
+const smallCheckboxSize = "16px";
+const mediumCheckboxSize = "20px";
+const largeCheckboxSize = "24px";
```

Comment:
I'd rather see tokens used here (except for the `20px` value).

Reply:
@mikrotron why no token for the 20px?